content/fr/docs/setup/release/building-from-source.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Builder une release
+content_template: templates/concept
+---
+{{% capture overview %}}
+Vous pouvez soit compiler une version à partir des sources, soit télécharger une version pré-compilée.  Si vous ne 
+prévoyez pas de développer Kubernetes nous vous suggérons d'utiliser une version pré-compilée de la version actuelle,
+ que l'on peut trouver dans le répertoire [Release Notes](/docs/setup/release/notes/).
+
+Le code source de Kubernetes peut être téléchargé sur le repo [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes).
+
+{{% /capture %}}
+
+{{% capture body %}}
+## Builder à partir des sources
+
+Si vous buildez simplement une version à partir des sources, il n'est pas nécessaire de mettre en place un environnement golang complet car tous les builds se font dans un conteneur Docker.
+
+Builder une version est simple.
```

Comment:
Alternative "Builder" : constituer, compiler etc

Reply:
i changed with "installer" or "construire" it depends

Reply:
is that ok for you guys?

Reply:
ok